### PR TITLE
Simplify LinuxWindowCapture

### PR DESCRIPTION
**Pull request by angrylinuxuser**
-Use QX11Info class to obtain xcb connection ( this adds QX11Extras as
dependency )
-Remove redundant headers.
-Update travis config to reflect changes.
-Update README to reflect changes.
-Use pkgconfig to obtain pkg info instead of hardcoded library locations

### CODE REVIEW

**Comment by BOSSoNe0013** [comment]
Thanks you @angrylinuxuser

File: src/LinuxWindowCapture.cpp [hunk 1]
```diff
@@ -70,103 +69,70 @@ bool LinuxWindowCapture::WindowRect() {
...
+                    translateR = xcb_translate_coordinates_reply( xcbConn, translateC, NULL );
+    if ( !translateR ) return false;
 
-    return true;
+    mRect.setTopLeft( QPoint( translateR->dst_x, translateR->dst_y ) );
+    free( translateR );
   }
-  xcb_disconnect( dpy );
-  return false;
+  return true;
```
Comment (MannyC) [inline src/LinuxWindowCapture.cpp]: treeR won't be freed unless the if(parent) path is taken

Comment (angrylinuxuser) [inline src/LinuxWindowCapture.cpp]: ups :) how did i missed that... ill fix this with my next pull :)

Comment (BOSSoNe0013) [inline src/LinuxWindowCapture.cpp]: fixed, thanks for the feedback :)

**Pull request merged**